lib/smart_app_launch/client_suite/token_request_alca_verification_test.rb

```diff
@@ -17,6 +17,21 @@ class SMARTClientTokenRequestAppLaunchConfidentialAsymmetricVerification < Infer
     description %(
       Check that SMART token requests are conformant.
     )
+    verifies_requirements 'hl7.fhir.uv.smart-app-launch_2.2.0@64',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@68',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@69',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@70',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@233',
+                          'hl7.fhir.uv.smart-app-launch_2.2.0@237',
```

Comment:
@237 is for backend services. remove from here even though it's technically checked. (@karlnaden should it be?)

@237
[when requesting] an access token via HTTP POST to the FHIR authorization server’s token endpoint URL [the] `client_assertion_type` [parameter is] `required` [and SHALL contain] … Fixed value: `urn:ietf:params:oauth:client-assertion-type:jwt-bearer`

Reply:
Removed in [9c54a3e](https://github.com/inferno-framework/smart-app-launch-test-kit/pull/105/commits/9c54a3e789e809d63025899a3073396314a4bd7b)